superset/utils/decorators.py

```diff
@@ -116,3 +118,20 @@ def wrapper(*args, **kwargs):
         return wrapper
 
     return decorator
+
+
+def logged_in_api(f: Callable) -> Callable:
+    """
+        Use this decorator to require a user to be logged to use your API methods.
```

Comment:
Nit. Can we use the docstring style suggested in [CONTRIBUTING.md](https://github.com/apache/incubator-superset/blob/master/CONTRIBUTING.md#typing).

Reply:
Done.

Reply:
Sorry could you also dedent so everything is left justified with the `"""`.

Reply:
Done.